books/RayTracingInOneWeekend.html

```diff
@@ -952,10 +952,41 @@
     [Listing [hittable-list-initial]: <kbd>[hittable_list.h]</kbd> The hittable_list class]
 </div>
 
-If you're unfamiliar with C++'s `std::vector`, this is a generic array-like collection of an
-arbitrary type. Above, we use a collection of pointers to `hittable`. `std::vector` automatically
-grows as more values are added: `objects.push_back(object)` adds a value to the end of the
-`std::vector` member variable `objects`.
+## Some New C++ Features
```

Comment:
Should probably qualify that `shared_ptr` is in the `<memory>` header

Reply:
Will do.